archey/logos/elementary.py

```diff
@@ -0,0 +1,24 @@
+"""Elementary OS logo"""
+
+ELEMENTARY = [
```

Comment:
Hey @SomethingGeneric ! Thanks for this.
One question though, have you tried an _empty_ logo instead of a _filled_ one ?

Bye 👋

Reply:
I'm not an artistic person, and the tool i found for ascii art filled the space with characters. I'm totally open to someone who's good at art doing an empty one.

Reply:
Oops sorry I edited my previous comment not fast enough.
On the same idea, ScreenFetch bundles an _empty_ one too : <https://github.com/KittyKatt/screenFetch/blob/42563a1de9750b02b5f6913ded76460445fe364a/screenfetch-dev#L4897>
What do you think about them ?

Reply:
That's awesome! If we're *allowed* to snatch it, then lets' go right ahead!